include/odp/api/spec/init.h

```diff
@@ -106,6 +106,26 @@ typedef int (*odp_log_func_t)(odp_log_level_t level, const char *fmt, ...);
 /** Replaceable abort function */
 typedef void (*odp_abort_func_t)(void) ODP_NORETURN;
 
+/**
+ * Application memory model
+ */
+typedef enum {
+	/** All memory is shareable between threads. All ODP handles and
+	 *  pointers to ODP allocated data may be shared amongst threads
+	 *  independent of data allocation time. */
+	ODP_MEM_MODEL_THREAD = 0,
+
+	/** By default all memory is not shareable between processes. All ODP
+	 *  handles and pointers to ODP allocated data (excluding non-single VA
+	 *  SHM blocks) may be shared amongst processes independent of data
+	 *  allocation time.
```

Comment:
... independent of data allocation time (e.g. before or after fork of a thread).

Reply:
If we want to clarify this, I'd clarify it to point out that sharing is only for threads/processes created after the `odp_init_global()` call, i.e., within a given ODP instance.